src/Core/Managed/Shared/Extensibility/Implementation/TaskTimer.cs

```diff
@@ -66,7 +66,11 @@ public void Start(Func<Task> elapsed)
 
             TaskEx.Delay(this.Delay, newTokenSource.Token)
                 .ContinueWith(
-                    async previousTask => 
+#if !NET40
```

Comment:
I have reservations about this entire method's existence, but the translation seems correct. :)